test/bun.js/websocket-server.test.ts

```diff
@@ -654,14 +653,14 @@ describe("websocket server", () => {
       fetch(req, server) {
         if (
           server.upgrade(req, {
-            count: 0,
+            data: { count: 0 },
           })
         )
           return new Response("noooooo hello world");
       },
     });
 
-    await new Promise((resolve, reject) => {
+    await new Promise<void>((resolve, reject) => {
```

Comment:
I don't think it's worth developer time for us to put `<void>` in here

Maybe we could enable noImplicitAny?

Reply:
`noImplicitAny` isn't the problem here - this is necessary based on the typing of `Promise` in the `lib.esnext` typings. I don't think there's a setting in tsconfig that would make this valid.

<img width="604" alt="Screenshot 2022-11-09 at 1 39 40 PM" src="https://user-images.githubusercontent.com/3084745/200961426-f4c54db2-74a0-4daa-9d45-ea34b65593df.png">